style-guide/front.html

```diff
@@ -97,22 +92,18 @@
 	<link rel='stylesheet' id='spine-theme-css' href='https://hub.wsu.edu/jeremy/wp-content/themes/spine/style.css?ver=0.26.16-1.5.1-38560' type='text/css' media='all' />
 	<link rel='stylesheet' id='spine-theme-print-css' href='https://hub.wsu.edu/jeremy/wp-content/themes/spine/css/print.css?ver=0.26.16-1.5.1-38560' type='text/css' media='print' />
 	<link rel="stylesheet" id="child-theme-css" href="../style.css" type="text/css" media="all" />
-
```

Comment:
Some of this spacing was intentional for readability. It helped (IMO) split up the sections of header info.